src/coreclr/src/inc/warningcontrol.h

```diff
@@ -23,11 +23,6 @@
 #pragma warning(3       :4132)   // const object should be initialized
```

Comment:
Why have you left some of the pragmas here? I've thought we will delete this file after moving the warnings.

Reply:
Oops, I misunderstood the issue! I thought only the `#pragma warning` ones were to be moved. Let me get working on the rest then.